### DISCUSSION TITLE
Where are the "options" props for DropDown and ComboBox  in V9

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [microsoft/fluentui#26105] [Q&A] [answered]
will  DropDown and ComboBox  in V9 ever have  "options" props ?

Dose it always be a SomeArray.map of <option > ?

**Comment by smhigley** [comment, marked as answer]
The v9 ones don't take options props since the options are defined as children, using the `<Option>` component. It would be pretty simple to extend the v9 Combo or Dropdown to take an `options` prop instead of children with something like this:

```tsx
type CustomOptionType = {
  key: string;
  text: string;
  // any other data
};

type CustomComboboxProps = ComboboxProps & {
  children: never;
  options: CustomOptionType[];
};

const CustomCombobox: ForwardRefComponent<ComboboxProps> = React.forwardRef((props, ref) => {
  return (
    <Combobox>
      {props.options.map((option) => {
        <Option key={option.key}>{option.text}</Option>
      })}
    </Combobox>
  );
});
```

**Reply by Ofer-Gal** [reply]
Thanks
`const CustomCombobox: ForwardRefComponent<ComboboxProps> = React.forwardRef((props, ref) => {`
should be
`const CustomCombobox: ForwardRefComponent<CustomComboboxProps> = React.forwardRef((props, ref) => {`

**Reply by smhigley** [reply]
yup, good catch!

**Reply by Dan86de** [reply]
Another question. Do we have any options to change default ListBox render container? Currently it's starts from input itself. Can we play with size and start point of rendering ListBox?

![image](https://user-images.githubusercontent.com/23312063/214065208-470a9f2e-71ed-4aed-bfe9-6e5ec832df37.png)
